docs/01-Using-Fleet/03-REST-API.md

```diff
@@ -607,18 +607,31 @@ Returns the count of all hosts organized by status. `online_count` includes all
 
 #### Parameters
 
-None.
+| Name    | Type    | In    | Description                                                                     |
+| ------- | ------- | ----  | ------------------------------------------------------------------------------- |
+| team_id | integer | query | The ID of the team whose host counts should be included. Defaults to all teams. |
 
 #### Example
 
-`GET /api/v1/fleet/host_summary`
+`GET /api/v1/fleet/host_summary?team_id=1`
 
 ##### Default response
 
 `Status: 200`
 
 ```json
 {
+  "totals_hosts_count": 2408,
```

Comment:
Should the requested team id (if any) be included in the response? E.g. `"team_id": 1` added when one is specified, omitted otherwise?

Reply:
@lukeheath thoughts?

Reply:
I'll merge as-is for now and prepare a new PR if we want to add this.

Reply:
Yes, let's include the id in the response any time a resource is requested by id.